wildfly-jakartaee-rest-archetype/src/main/resources/archetype-resources/pom.xml

```diff
@@ -0,0 +1,213 @@
+
+        <!-- other plugin versions -->
+        <version.compiler.plugin>3.8.0</version.compiler.plugin>
+        <version.surefire.plugin>2.22.1</version.surefire.plugin>
+        <version.failsafe.plugin>2.22.1</version.failsafe.plugin>
+        <version.war.plugin>3.2.2</version.war.plugin>
+
+        <!-- maven-compiler-plugin -->
+        <maven.compiler.target>1.8</maven.compiler.target>
+        <maven.compiler.source>1.8</maven.compiler.source>
+    </properties>
+    
+    <dependencyManagement>
+        <dependencies>
+            <!-- JBoss distributes a complete set of Jakarta EE 8 APIs including
+                a Bill of Materials (BOM). A BOM specifies the versions of a "stack" (or 
+                a collection) of artifacts. We use this here so that we always get the correct 
+                versions of artifacts. Here we use the wildfly-jakartaee-8.0-with-tools stack
+                (you can read this as the WildFly stack of the Jakarta EE 8 APIs, with some extras
+                tools for your project, such as Arquillian for testing) -->
+            <dependency>
+                <groupId>org.wildfly.bom</groupId>
+                <artifactId>wildfly-jakartaee8-with-tools</artifactId>
+                <version>${version.jboss.bom}</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
+        </dependencies>
+    </dependencyManagement>
+	
+    <dependencies>
+
+        <!-- Import the CDI API, we use provided scope as the API is included in JBoss EAP -->
+        <dependency>
+            <groupId>jakarta.enterprise</groupId>
+            <artifactId>jakarta.enterprise.cdi-api</artifactId>
+            <scope>provided</scope>
+        </dependency>
+
+        <!-- Import the JAX-RS API, we use provided scope as the API is included in JBoss EAP -->
+        <dependency>
+            <groupId>org.jboss.spec.javax.ws.rs</groupId>
+            <artifactId>jboss-jaxrs-api_2.1_spec</artifactId>
+            <scope>provided</scope>
+        </dependency>
+        
+        <!--dependencies for MicroProfile APIs -->
+        
+
+        <!-- Test scope dependencies -->
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <scope>test</scope>
+        </dependency>
+        
+    </dependencies>
+	
+    <build>
+        <!-- Tell Maven that the resulting file should not have a file name containing the version - 
+			 a non versioned name is required e.g. when building a deployable artifact using the ShrinkWrap API -->
+		<finalName>${project.artifactId}</finalName>
+
+        <plugins>
+			
+			<!--Build configuration for the WAR plugin: -->
+			<plugin>
+				<artifactId>maven-war-plugin</artifactId>
+				<version>${version.war.plugin}</version>
+				<configuration>
+					<!-- Jakarta EE doesn't require web.xml, Maven needs to catch up! -->
+					<failOnMissingWebXml>false</failOnMissingWebXml>
```

Comment:
do we need to define maven-war-plugin? This property can be defined in the properties block I believe.